src/tool_launcher.hpp

```diff
@@ -260,6 +260,8 @@ private Q_SLOTS:
 
 	bool m_useNativeDialogs;
 
+	bool m_dac_tools_failed, m_adc_tools_failed;
```

Comment:
are this values initialised in the constructor of the ToolLauncher?

Reply:
I added the initialization now.